design/README.md

```diff
@@ -0,0 +1,32 @@
+# HackWPI 2018 Design
+Design documents for HackWPI 2018
+
+## Preview
+
+![Home](Home.jpg)
+
+![Day of](Day-of.jpg)
+
+![Map](Map.jpg)
+
+## Editing
+
+You need a Mac with [Sketch](https://www.sketchapp.com) installed to open the document.
+
+## Contributing
```

Comment:
Should this be in the top level readme?

Reply:
Talked this over with Harry, this secion is specifically for the design documents.